RFC 5706 versus RFC5706bis different audiences #37 (#71)
IETF-OPSAWG-WG/draft-opsarea-rfc5706bis: draft-opsarea-rfc5706bis.md

   workshop, the NEMOPS workshop reviewed the past outcomes and discussed any
   operational barriers that prevented these technologies from being widely
   implemented. With the industry, network operators, and protocol engineers
   working in collaboration, the workshop developed a proposed plan of action
   and provided network management recommendations for both the IETF and IRTF.
   See {{I-D.iab-nemops-workshop-report}} for the detailed information.

   As a matter of fact, the operational environment and manageability of a protocol should
   be considered from the start when New Protocols are designed.

##  This Document {#sec-this-doc}

   This document provides some initial guidelines for considering
   operations and management in an IETF management framework that
   consists of multiple protocols and multiple data modeling languages,
   with an eye toward being flexible while also striving for
   interoperability.

   Fully New Protocols may require significant consideration of expected
   operations and management, while extensions to existing, widely
   deployed protocols may have established de facto operations and

>    management practices that are already well understood.

Reviewer: ```suggestion
   management practices that are already well understood. However, the guidance
   in this document does not mandate providing a comprehensive inventory of
   all operational and manageability considerations. Instead, the guidance is to
   focus on key aspects that are will ease deployability and are key to operate and maintain
   the technology. 
```

   management technology, such as how to utilize management protocols
   and how to design management data models.

##  Audience {#sec-audience}

   The purpose of this document is to provide guidance about what to
   consider when thinking about the management and deployment of a new
   protocol, and to provide guidance about documenting the
   considerations. As it is useful to anyone involved in the document lifecyle:
   from the authors drafting the protocol to those reviewing and evaluating its draft

>    content, including the WG chair, WG advisor, Document Shepherd, responsible AD, and the IESG.

Reviewer 2: Why is "Document Shepherd" capitalized, when all the others are not?

>    content, including the WG chair, WG advisor, Document Shepherd, responsible AD, and the IESG.

Reviewer: because I was sure that one is consistent with our conventions (RFC4858). For the others, I wasn't sure about the conventions out there.

   Protocol Designers should avoid having manageability pushed for a later
   phase of the development of the standard.

   When a WG considers operation and management functionality for a
   protocol, the document should contain enough information for readers
   to understand how the protocol will be deployed and managed. The considerations
   do not need to be comprehensive and exhaustive; focus should be on key aspects. The WG
   should expect that considerations for operations and management may
   need to be updated in the future, after further operational
   experience has been gained.

   For the OPS Area Directors or the IESG, this document helps evaluating the
   content of the new "Operational and Management Considerations" section. As an
   Area Director busy creating a new charter, this document document some considerations
   of the functionality needed to operate and manage new Protocol and Protocol Extensions.

>    The OPS directorate can use this document to guide the review. On top of that, a list of guidelines and a

Reviewer: ```suggestion
   The OPS directorate can use this document to guide performing reviews. On top of that, a list of guidelines and a
```

   checklist of questions to consider, which a reviewer can use to evaluate whether the protocol and
   documentation address common operations and management needs, is provided in {{CHECKLIST}}.

   This document is also of interest to the broader community, who wants to understand, contribute to,

>    and review drafts, taking into OPS considerations into account.

Reviewer: ```suggestion
   and review Internet-Drafts, taking into OPS considerations into account.
```
